JuliaGPU/session2/Manav/Q1.jl

```diff
@@ -0,0 +1,28 @@
+function Quicksort!(arr,low,high)
+	function Partition!(arr,low,high)
+   		i=low-1
+  		pivot=arr[high]
+    		for j =low:(high-1)
+        		if arr[j]<=pivot
+            			i=i+1
+            			arr[i],arr[j]=arr[j],arr[i]
+			end
+		end
+        	arr[i+1],arr[high]=arr[high],arr[i+1]
+    		return(i+1)
+	end
+	if low<high
+		pivot=Partition!(arr,low,high)
+    		Quicksort!(arr,low,pivot)
+    		Quicksort!(arr,pivot+1,high)
```

Comment:
There is an error here in how you recursively call Quicksort!

Reply:
OK shall resolve it and resend...